lib/flutter/flutter_daemon.dart

```diff
@@ -393,6 +408,22 @@ class DeviceDomain extends Domain {
   Future enable() => _call('device.enable');
 
   Future disable() => _call('device.disable');
+
+  Future<int> forward(String deviceId, int devicePort, int hostPort) {
+    return _call('device.forward', _stripNullValues({
+      'deviceId': deviceId,
+      'devicePort': devicePort,
+      'hostPort': hostPort,
+    })).then((Map<String, dynamic> result) => result['hostPort']);
+  }
+
+  Future unforward(String deviceId, int devicePort, int hostPort) {
```

Comment:
The DevicePortForwarder.unforward interface in flutter_tools takes a ForwardedPort that includes both device and host ports.  So an implementation might need the host port.